rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantCreator.java

```diff
@@ -16,23 +16,28 @@
 
   Participant from(ParticipantDTO participantDTO) {
     final Participant.ParticipantBuilder participantBuilder = Participant.builder()
-        .id(participantDTO.getId())
-        .firstName(participantDTO.getFirstName())
-        .lastName(participantDTO.getLastName())
-        .pesel(participantDTO.getPesel())
-        .parishId(participantDTO.getParishId());
-
-    final Address address = fromValue(participantDTO.getAddress());
-    participantBuilder.address(address);
+        .id(participantDTO.getId());
 
     final PersonalDataValue personalData = participantDTO.getPersonalData();
+    final Address address = fromValue(personalData.getAddress());
+
     participantBuilder
+        .firstName(personalData.getFirstName())
+        .lastName(personalData.getLastName())
+        .pesel(personalData.getPesel())
+        .phoneNumber(personalData.getPhoneNumber())
+        .email(personalData.getEmail())
+        .parishId(personalData.getParishId())
+        .address(address)
         .motherName(personalData.getMotherName())
         .fatherName(personalData.getFatherName())
         .christeningDate(convertToDateTime(personalData.getChristeningDate()))
         .christeningPlace(personalData.getChristeningPlace())
         .closeRelativeName(personalData.getEmergencyContactName())
-        .closeRelativeNumber(personalData.getEmergencyContactNumber());
+        .closeRelativeNumber(personalData.getEmergencyContactNumber())
```

Comment:
Niespójność: closeRelative / emergencyContact

Reply:
Jest już na to issue, uporzadkuje to w oddzielnym PR.